pallets/template/src/lib.rs

```diff
@@ -113,5 +125,21 @@ pub mod pallet {
 
 			Ok(book_id)
 		}
+
+		#[transactional]
+		pub fn transfer_book_to(book_id: &T::Hash, to: &T::AccountId) -> Result<(), Error<T>> {
+			let mut book = Self::books(&book_id).ok_or(<Error<T>>::BookNotExist)?;
+
+			let prev_owner = book.owner.clone();
+
+			// Remove book_id from BooksOwned of previous book owner
```

Comment:
Can you help to implement this method? with

https://docs.substrate.io/rustdocs/latest/frame_support/storage/trait.StorageValue.html#tymethod.try_mutate

Reply:
I'm following this tutorial: https://docs.substrate.io/tutorials/v3/kitties/pt1/